ocfweb/main/templates/home.html

```diff
@@ -26,7 +26,14 @@
 {% block content %}
 <div class="row">
     <div class="col-sm-8 ocf-content-block">
+
         {{lab_status.banner_html | safe}}
+
+        <a href="{% url 'doc' 'services/lab' %}" class="update-banner">
+            <h4>Lab hours have been extended!</h4>
+            <p>Click here to see the updated hours.</p>
+        </a>
+
```

Comment:
I was thinking to save that for urgent messages ("printing is on fire") rather than periodic updates but open to suggestions.

For example, we'll want to post a message long-term when a move date is set, this seemed a good use for that.